### DISCUSSION TITLE
Wildcard SSL second level subdomains

### DISCUSSION THREAD

**Discussion by jshapiro2** [postalserver/postal#2552] [Help with using Postal]
Hello,

Wildcard SSL certs can not be used with second level subdomains.  This appears to cause issues when you are using a subdomain for the postal server and then applying all the second level domains to the dns part. 
For example the server is on mail.domain.com
DNS from postal.yml
  mx_records:
    - mx.mail.domain.com
  smtp_server_hostname: mail.domain.com
  spf_include: spf.mail.domain.com
  return_path: rp.mail.domain.com
  route_domain: routes.mail.domain.com
  track_domain: track.mail.domain.com

For tracking of clicks and opens, can I just change track.mail.domain.com to track1.domain.com?  If I do this what else has to be done to make it work?

**Comment by willpower232** [comment]
I think you're confusing the domains.

The domains listed in the config file are for referring to Postal itself.

You would send email from a different domain which you configure via the Postal UI.

You would add a tracking subdomain via the Postal UI which is also different to the domains in the config file.

You would need any valid certificate to be set up for the tracking subdomain according to whatever proxy (Caddy or nginx etc) you are using.
